packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx

```diff
@@ -83,11 +91,17 @@ const PrintIdentifierSticker: React.FC<PrintIdentifierStickerProps> = ({ closeMo
         <div ref={contentToPrintRef}>
           <style type="text/css" media="print">
             {`
-              @page {
-                size: ${pageSize};
-              }
-            `}
+        @page {
+          size: ${pageSize};
+        }
+      `}
```

Comment:
Fix this formatting issue

Reply:
@jnsereko Thanks for the review. I have fixed.